source/agora/node/Ledger.d

```diff
@@ -576,7 +578,7 @@ unittest
         Transaction[] transactions;
 
         // always use the same amount, for simplicity
-        const Amount AmountPerTx = Amount(40_000);
+        const Amount AmountPerTx = Amount(400_000_000_000L);
```

Comment:
Once the freeze amount is added to the `Amount` struct, it can be used here directly.